__PROJECT_NAME__/.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: objective-c
+osx_image: xcode8.2
+env:
+  global:
+  - LC_CTYPE=en_US.UTF-8
+  - LANG=en_US.UTF-8
+  - WORKSPACE="__PROJECT_NAME__.xcworkspace"
+  - IOS_FRAMEWORK_SCHEME="__PROJECT_NAME__"
+  - IOS_SDK=iphonesimulator10.2
+  - EXAMPLE_SCHEME="iOS Example"
+  matrix:
+   - DESTINATION="platform=iOS Simulator,OS=10.2,id=E40727B3-41FB-4D6E-B4CB-BFA87109EB12"  SCHEME="$IOS_FRAMEWORK_SCHEME"      SDK="$IOS_SDK"      POD_LINT="NO"
+script:
+  - set -o pipefail
+  - xcodebuild -version
+  - xcodebuild -showsdks
+  - xcodebuild -list
+  - xcodebuild -workspace "$WORKSPACE" -list
+
+  # Build and test Framework in Debug
+  - xcodebuild -workspace "$WORKSPACE" -scheme "$SCHEME" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO test | xcpretty -c
+
+  # Build Framework in ReleaseTest
+  - xcodebuild -workspace "$WORKSPACE" -scheme "$SCHEME" -sdk "$SDK" -destination "$DESTINATION" -configuration Release ONLY_ACTIVE_ARCH=NO build | xcpretty -c
+
+  # Build Example in Debug if specified
+  - xcodebuild -workspace "$WORKSPACE" -scheme "$EXAMPLE_SCHEME" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO build | xcpretty -c
+
+  # Run `pod lib lint` if specified
+  - if [ $POD_LINT == "YES" ]; then
+      pod lib lint;
+    fi
+
+  - CheckCocoaPodsQualityIndexes.rb __PROJECT_NAME__
```

Comment:
I think this just adds noise, it should probably go, most people are fine to check the website